### ISSUE TITLE
Urban terror on raspberry pi

### ISSUE THREAD

**Issue by symbios24** [pedrxd/MinModUrT#5]
hello is it possible to install urban terror in raspberry pi?

i cannot find any instruction to do so anywhere

**Comment by pedrxd** [comment]
Well, quake is compatible with arm. Probably you can run it, but you need found a proper binary. 
[You can try with this](https://packages.debian.org/sid/ioquake3)

**Comment by pedrxd** [comment]
I prepare some containers multiarch for run UrbanTerror (server version). 
[https://hub.docker.com/r/pedrxd/urbanterror](https://hub.docker.com/r/pedrxd/urbanterror)
After some testing on raspberry, it seem to have very poor performance. Max 4 players.

**Comment by symbios24** [comment]
Hello.how did you install urban terror in the Raspberry Pi ??

**Comment by pedrxd** [comment]
Install docker and follow instructions [here](https://hub.docker.com/r/pedrxd/urbanterror). This is for server, not for client

**Comment by symbios24** [comment]
Yes but I don't want server but client.are there any instructions for it?

**Comment by pedrxd** [comment]
1. Execute: apt-get install gcc make libsdl2-dev
2. Download this [zip](https://github.com/mickael9/ioq3/archive/urt.zip) and unzip it
3. Open Makefile and set the line where USE_AUTH=1 to USE_AUTH=0
4. Execute: make -j4 
5. Go to build/release-*-* and copy the two *.so files and the urbanterror-m9 to the urbanterror43 folder. 
6. Execute urbanterror-m9 file on the urbanterror43 folder

It "should" work. I don't test it
